Samples/Forms/LibVLCSharp.Forms.Sample.GTK/LibVLCSharp.Forms.Sample.GTK.csproj

```diff
@@ -0,0 +1,157 @@
+    <Reference Include="webkit-sharp, Version=1.1.15.0, Culture=neutral, PublicKeyToken=eaa1d335d2e19745, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Xamarin.Forms.Platform.GTK.3.2.0.871581\lib\net45\webkit-sharp.dll</HintPath>
+    </Reference>
+    <Reference Include="Xamarin.Forms.Core, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Xamarin.Forms.3.2.0.871581\lib\netstandard2.0\Xamarin.Forms.Core.dll</HintPath>
+    </Reference>
+    <Reference Include="Xamarin.Forms.Platform, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Xamarin.Forms.3.2.0.871581\lib\netstandard2.0\Xamarin.Forms.Platform.dll</HintPath>
+    </Reference>
+    <Reference Include="Xamarin.Forms.Platform.GTK, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Xamarin.Forms.Platform.GTK.3.2.0.871581\lib\net45\Xamarin.Forms.Platform.GTK.dll</HintPath>
+    </Reference>
+    <Reference Include="Xamarin.Forms.Xaml, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Xamarin.Forms.3.2.0.871581\lib\netstandard2.0\Xamarin.Forms.Xaml.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Program.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="App.config" />
+    <None Include="OpenTK.dll.config" />
+    <None Include="packages.config" />
+    <None Include="webkit-sharp.dll.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\LibVLCSharp.Forms.Platforms.Gtk\LibVLCSharp.Forms.Platforms.GTK.csproj">
+      <Project>{3F958392-C1A0-407C-AC76-0926F8D0BCCA}</Project>
+      <Name>LibVLCSharp.Forms.Platforms.GTK</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\..\LibVLCSharp.Forms\LibVLCSharp.Forms.csproj">
+      <Project>{06814458-c8b4-4d50-86e2-904e6a9e3499}</Project>
+      <Name>LibVLCSharp.Forms</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\..\LibVLCSharp\LibVLCSharp.csproj">
+      <Project>{d1c3b7c4-713b-46b2-b33a-e9298c819921}</Project>
+      <Name>LibVLCSharp</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\LibVLCSharp.Forms.Sample\LibVLCSharp.Forms.Sample.csproj">
+      <Project>{a7f1b21f-ca82-4fc7-871f-2be5b9d6da70}</Project>
+      <Name>LibVLCSharp.Forms.Sample</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+    <PropertyGroup>
+      <ErrorText>Ce projet fait référence à des packages NuGet qui sont manquants sur cet ordinateur. Utilisez l'option de restauration des packages NuGet pour les télécharger. Pour plus d'informations, consultez http://go.microsoft.com/fwlink/?LinkID=322105. Le fichier manquant est : {0}.</ErrorText>
```

Comment:
![](https://www.theflagshop.co.uk/media/catalog/product/cache/1/thumbnail/9df78eab33525d08d6e5fb8d27136e95/f/r/france-flag-std_2.jpg)

Reply:
What's the english version of those messages?

Reply:
https://github.com/videolan/libvlcsharp/blob/9cf2464b623457930513a2799e1c3d4a9a6292b3/Samples/LibVLCSharp.Android.Sample/LibVLCSharp.Android.Sample.csproj#L102

I see most samples have it in french as well

Reply:
This project is now SDK-Style, so no problem anymore.